ubuntu-installer-classic-23.04-amd64.json

```diff
@@ -0,0 +1,41 @@
+{
+    "type": "model",
+    "series": "16",
+    "authority-id": "canonical",
+    "brand-id": "canonical",
+    "model": "ubuntu-installer-classic-23.04-amd64",
+    "architecture": "amd64",
+    "timestamp": "2023-03-19T12:00:00.0Z",
+    "base": "core22",
+    "grade": "signed",
+    "classic": "true",
+    "distribution": "ubuntu",
+    "snaps": [
+        {
+            "name": "core22",
+            "type": "base",
+            "default-channel": "latest/stable",
+            "id": "amcUKQILKXHHTlmSa7NMdnXSx02dNeeT"
+        },
+        {
+            "name": "snapd",
+            "type": "snapd",
+            "default-channel": "latest/stable",
+            "id": "PMrrV4ml8uWuEUDBT8dSGnKUYbevVhc4"
+        },
+        {
+            "name": "subiquity",
+            "type": "app",
+            "classic": "true",
+            "default-channel": "latest/stable",
```

Comment:
We are definitely not going to be tracking stable for subiquity here. Not sure what would be best -- probably some canary specific branch for now.